src/main/java/Duke.java

```diff
@@ -1,10 +1,268 @@
+            var correct = true;
+            var deleted = false;
+            var done = order.length() >= 4 && order.substring(0 , 4).equals("done");
+            var delete = order.length() >= 6 && order.substring(0 , 6).equals("delete");
+            var todo = order.length() >= 4 && order.substring(0 , 4).equals("todo");
+            var deadline = order.length() >= 8  && order.substring(0, 8).equals("deadline");
+            var event = order.length() >= 5 && order.substring(0, 5).equals("event");
+            if (order.equals("food left")) {
+                System.out.println("Ka To:");
+                System.out.println(" ");
+                System.out.println("Only an apple pie");
+            } else if (order.equals("allowance left")) {
+                System.out.println("Ka To:");
+                System.out.println(" ");
+                System.out.println("I have checked, it is 20000 SGD");
+            } else if (order.equals("who is the most handsome guy")) {
+                System.out.println("Ka To:");
+                System.out.println(" ");
+                System.out.println("Of Course You!");
+            } else if (order.equals("task list")) {
+                System.out.println("Ka To:");
+                System.out.println(" ");
+                if(orders.size() == 0){
+                    System.out.println(" You have not given me any task yet ");
+                }else {
+                    for (int i = 0; i < orders.size(); i++) {
+                        if(orders.get(i).status == 0) {
+                            if(orders.get(i).category == 0){
+                                System.out.println(i + 1 + ". " + " [T][x] "+ orders.get(i).command);
+                            }else if(orders.get(i).category == 1){
+                                System.out.println(i + 1 + ". " + " [D][x] " + orders.get(i).command);
+                            }else if(orders.get(i).category == 2){
+                                System.out.println(i + 1 + ". " + " [E][x] " + orders.get(i).command);
+                            }else {
+                                System.out.println(i + 1 + ". " + " [x] " + orders.get(i).command);
+                            }
+                        }else {
+                            if(orders.get(i).category == 0){
+                                System.out.println(i + 1 + ". " + " [T][v] "+ orders.get(i).command);
+                            }else if(orders.get(i).category == 1){
+                                System.out.println(i + 1 + ". " + " [D][v] " + orders.get(i).command);
+                            }else if(orders.get(i).category == 2){
+                                System.out.println(i + 1 + ". " + " [E][v] " + orders.get(i).command);
+                            }else {
+                                System.out.println(i + 1 + ". " + " [v] " + orders.get(i).command);
+                            }
+                        }
```

Comment:
One possible way to make the printing of the tasks out neater could be by overriding the toString method of your Task class so that you can just call System.out.println(i + 1 + ". " + orders.get(i)) instead. It is the same logic as what you are doing here, just that the checking of status and category is done inside the Task class and this will help with abstraction.